test/strictMode/output/umd/emptyImport.js

```diff
@@ -1,7 +1,7 @@
 (function (global, factory) {
-	typeof exports === 'object' && typeof module !== 'undefined' ? factory(require('foo'), require('polyfills')) :
-	typeof define === 'function' && define.amd ? define(['foo', 'polyfills'], factory) :
-	factory(global.bar)
-}(this, function (bar) { 'use strict';
+	typeof exports === 'object' && typeof module !== 'undefined' ? factory(require('polyfills'), require('foo')) :
+	typeof define === 'function' && define.amd ? define(['polyfills', 'foo'], factory) :
+	factory(global.__dep0__, global.bar)
```

Comment:
Sure. In this UMD case that seems good.

For AMD I don't think we can stop the loader from passing the value of the module, but that's probably ok.